src/main/java/com/exasol/dbbuilder/dialects/mysql/MySqlSchema.java

```diff
@@ -0,0 +1,29 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import com.exasol.dbbuilder.dialects.*;
+
+/**
+ * MySQL database schema.
+ */
+public class MySqlSchema extends AbstractSchema {
```

Comment:
This looks like 100% code duplication. We should be able to fix that with the writer pull-up and replacing the `QuoteApplier`.

Reply:
done